### PR TITLE
DotPulse application

### PR DESCRIPTION
**Pull request by arctic-ash**
#### Project Abstract

> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [x] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/grant_guidelines_per_category.md).
- [x] A BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) address for the payment of the milestones is provided inside the application.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by arctic-ash** [comment]
Thanks @Noc2 for your fast reply. I made the required changes.

**Review by Noc2 — APPROVED** [review]
Thanks for the update. I’m happy to go ahead with it and mark the application as ready for review.

**Comment by arctic-ash** [comment]
Thanks @Noc2!

**Review by gautamdhameja — DISMISSED** [review]

**Review by SBalaguer — APPROVED** [review]

**Comment by laboon** [comment]
I am unable to access any of the links provided for the Github contributors:

- https://github.com/orgs/CrossChainLabs/people/yellow-cube
- https://github.com/orgs/CrossChainLabs/people/aliasccl
- https://github.com/orgs/CrossChainLabs/people/arctic-ash

File: applications/DotPulse.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+- https://github.com/CrossChainLabs/audit-registry
+- https://github.com/CrossChainLabs/near-dns
+- https://github.com/CrossChainLabs/near-api-go
+- https://github.com/CrossChainLabs/coredns-near
+- https://github.com/CrossChainLabs/near-link
+- https://github.com/CrossChainLabs/filmarket-contract
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/orgs/CrossChainLabs/people/yellow-cube
```
Comment (laboon) [inline applications/DotPulse.md:82]: I am unable to access any of the links provided for the Github contributors:

https://github.com/orgs/CrossChainLabs/people/yellow-cube
https://github.com/orgs/CrossChainLabs/people/aliasccl
https://github.com/orgs/CrossChainLabs/people/arctic-ash

**Review by laboon — APPROVED** [review]

File: applications/DotPulse.md (hunk 1 above)
Comment (arctic-ash) [reply applications/DotPulse.md]: hey @laboon! I fixed the Github contributors' links.

**Review by Noc2 — DISMISSED** [review]

**Review by gautamdhameja — APPROVED** [review]

**Review by SBalaguer — APPROVED** [review]

**Review by takahser — CHANGES_REQUESTED** [review]
@arctic-ash thanks for the submission for your grant application. I've got a couple of questions here:

1. Could you add delivery _0e. Article_? It's [usually mandatory](https://github.com/w3f/Grants-Program/blob/d9640bdb1ea2176cfa86d90530a5e44b5276737f/applications/application-template.md#development-roadmap-nut_and_bolt) for at least 1 of the milestones.
2. It's not completely clear to me how you're planning to identify Polkadot/Dotsama/Substrate-related organisations and repositories on Github. You're stating that you want to track "the development among different Polkadot organisations on Github", so will it just be a list of static organisations that you'll track? Do you have any strategy to update it?
3. I like the set of features you're planning to implement and the structure of the Dashboard mockup makes sense to me. However, I'm a bit confused about the design - Are you aiming for a touch of [Neumorphism](https://www.justinmind.com/ui-design/neumorphism) here? Or are these elements supposed to be cards, such as in UI frameworks like [material design](https://material.io/components/cards), for example? In the latter case the shades would make more sense around the cards, rather than within the cards.

**Comment by arctic-ash** [comment]
> @arctic-ash thanks for the submission for your grant application. I've got a couple of questions here:
> 
> 1. Could you add delivery _0e. Article_? It's [usually mandatory](https://github.com/w3f/Grants-Program/blob/d9640bdb1ea2176cfa86d90530a5e44b5276737f/applications/application-template.md#development-roadmap-nut_and_bolt) for at least 1 of the milestones.

Done.

> 2. It's not completely clear to me how you're planning to identify Polkadot/Dotsama/Substrate-related organisations and repositories on Github. You're stating that you want to track "the development among different Polkadot organisations on Github", so will it just be a list of static organisations that you'll track? Do you have any strategy to update it?

Yes, it will be a list of static organisations that can be easily extended by adding new organisations in an environment variable. It would be useful to have an official list of Polkadot organisations if possible, to make sure that we include all of them. We’ll make the commitment to update the list as soon as we’re aware of the existence of new organisations.

> 3. I like the set of features you're planning to implement and the structure of the Dashboard mockup makes sense to me. However, I'm a bit confused about the design - Are you aiming for a touch of [Neumorphism](https://www.justinmind.com/ui-design/neumorphism) here? Or are these elements supposed to be cards, such as in UI frameworks like [material design](https://material.io/components/cards), for example? In the latter case the shades would make more sense around the cards, rather than within the cards.

Thanks for the appreciation! Those elements are supposed to be cards, our designer was a bit more creative but the frontend will be developed using the Material UI framework. So the shades will be on the outside, not within the cards.

**Review by Noc2 — APPROVED** [review]
As pointed out before, regarding the list, you might want to take a look at https://wiki.polkadot.network/docs/build-open-source Besides this, I’m still happy to go ahead with it.

**Comment by arctic-ash** [comment]
Thanks @Noc2, the wiki page is helpful indeed as we can build a comprehensive list of all the organisations.

**Review by laboon — APPROVED** [review]

**Review by takahser — APPROVED** [review]
@arctic-ash thanks for answering my questions, it makes all sense to me now. Happy to approve here.

**Review by SBalaguer — APPROVED** [review]

**Review by gautamdhameja — APPROVED** [review]

**Pull request merged**